utils/zerovec/src/map2d/mod.rs

```diff
@@ -6,6 +6,9 @@
 
 mod borrowed;
 pub(crate) mod map;
+
+#[cfg(feature = "crabbake")]
+mod crabbake;
```

Comment:
suggestion(nb): i'd be okay with all the zerovec types having crabbake impls in one file, the way we do for yoke. either works.

Reply:
I've been following serde because I think that's a better metaphor than yoke.